examples/makeInverseOperator/main.cpp

```diff
@@ -88,14 +89,34 @@ int main(int argc, char *argv[])
 {
     QCoreApplication a(argc, argv);
 
-    QFile t_fileFwdMeeg("./MNE-sample-data/MEG/sample/sample_audvis-meg-eeg-oct-6-fwd.fif");
-    QFile t_fileFwdEeg("./MNE-sample-data/MEG/sample/sample_audvis-eeg-oct-6-fwd.fif");
-    QFile t_fileCov("./MNE-sample-data/MEG/sample/sample_audvis-cov.fif");
-    QFile t_fileEvoked("./MNE-sample-data/MEG/sample/sample_audvis-ave.fif");
-
-    double snr = 3.0;
+    // Command Line Parser
+    QCommandLineParser parser;
+    parser.setApplicationDescription("Start makeInverseOperator example");
```

Comment:
Consider a more descriptive explaination isntead of the camelCase makeInverseOperator